api/public/public_api/api.py

```diff
@@ -16,7 +16,7 @@
 #
 # You should have received a copy of the GNU Affero General Public License
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
-
+"""Define the Selene Public API"""
```

Comment:
Looks like this file was just cleaned up to consistently use double rather than single quotes. If I am missing something else please let me know.

Reply:
There is a new route at the bottom of the file for the new endpoint.